pkg/virtual/framework/forwardingregistry/rest.go

```diff
@@ -57,26 +59,27 @@ func newStores(ctx context.Context, gvr schema.GroupVersionResource, dynamicClus
 				ret.SetGroupVersionKind(listKind)
 				return ret
 			},
-			DefaultQualifiedResource: resource,
-			CreateStrategy:           strategy,
-			UpdateStrategy:           strategy,
-			DeleteStrategy:           strategy,
-			TableConvertor:           tableConvertor,
-			ResetFieldsStrategy:      strategy,
+			CreateStrategy:      strategy,
+			UpdateStrategy:      strategy,
+			DeleteStrategy:      strategy,
+			TableConvertor:      tableConvertor,
+			ResetFieldsStrategy: strategy,
 
 			resource:                  gvr,
 			dynamicClusterClient:      dynamicClusterClient,
 			patchConflictRetryBackoff: *patchConflictRetryBackoff,
-			labelSelector:             labelSelector,
-
-			stopWatchesCh: ctx.Done(),
+			stopWatchesCh:             ctx.Done(),
 		}
+		store := wrapper(resource, delegate)
```

Comment:
Should wrapper be optional?

Reply:
We don't currently have a use-case for it, so if someone comes up with one we can merge the changes required to support it.

Reply:
The APIExport virtual workspace doesn't need it?

Reply:
I can put it in my PR if needed